008-strimzi-ui.md

```diff
@@ -0,0 +1,119 @@
+# Add a UI to Strimzi for Kafka Administration
+
+This proposal is to add a Kafka administration UI to the Strimzi project.
+
+## Motivation
+
+Kafka out of the box provides a number of useful command tools to perform common administrative tasks such as creating topics, viewing consumer group status, and consuming record content. While these are helpful, they are not always the simplest or most convenient way of monitoring and performing administrative tasks on a Kafka cluster. In these cases a dedicated administration UI, focused on common administration tasks, would not only enable a user to complete their task more quickly and simply than previously possible, but also enable a more positive user experience of Kafka and its capabilities.
+
+## Proposal
+
+This proposal will describe a Kafka administration UI capability. It will describe example capabilities which could be offered, cover at a high level how the UI code itself could be implemented (with more low level detail to be provided as required), how this could be delivered, as well as possible future work and how this could integrate with Strimzi today.
+
+### UI implementation
+
+At a high level, I would propose a UI implemented as follows:
+
+- It would be a [React](https://reactjs.org/) based UI
```

Comment:
I know it might be heavily implied, but it might make sense to clarify that it is written mostly in Javascript due to its dependency on React?

Reply:
Can do - I see no harm in being explicit.